meson_options.txt

```
@@ -25,3 +25,4 @@ option('enable-tizen-sensor', type: 'boolean', value: false)
 option('enable-edgetpu', type: 'boolean', value: false)
 option('enable-armnn', type: 'boolean', value: false)
 option('enable-openvino', type: 'boolean', value: false)
+option('framework-priority-tflite', type: 'string', value: 'tensorflow-lite,nnfw,armnn,edgetpu', description: 'A comma separated prioritized list of neural network frameworks to open .tflite file')
```

This is a just suggestion. We've already used the name to indicate 'TensorFlow Lite' as 'tensorflow-lite' in the meson_options.txt file. For the consistency, how about 'framework-priority-tensorflow-lite' (I know it is quite long):(
Anyway, I think we need to use the keyword, 'tensorflow-lite', at least in this file.

@myungjoo PTAL:)

---

In my opinion, 'framework-priority-tflite' represents an extension '.tflite' not tensorflow-lite framework, so it does not seem necessary to change it.

---

You are right:)

```suggestion
option('framework-priority-tflite', type: 'string', value: 'tensorflow-lite,nnfw,armnn,edgetpu', description: 'A comma separated prioritized list of neural network frameworks to open a .tflite file')
```

or 

```suggestion
option('framework-priority-tflite', type: 'string', value: 'tensorflow-lite,nnfw,armnn,edgetpu', description: 'A comma separated prioritized list of neural network frameworks to open a given .tflite file')
```

---

Thank you! Updated with above suggestion.